go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/hirakiuc/ec2s
+
+go 1.18
+
+require (
+	github.com/BurntSushi/toml v0.3.0
+	github.com/aws/aws-sdk-go v1.8.22-0.20170509180017-2a4ef28e5761
```

Comment:
This part is the most riskiest change in this PR because I'm not completely sure whether this version is compatible with the previous one which this app depends on. 👀 

Additionally, I have no actual AWS environment to test this change just now. 😭 

In any case, I'll merge this change to a topic branch for further improvements. 🙏🏼